src/libs/actions/connections/QuickbooksOnline.ts

```diff
@@ -86,6 +86,87 @@ function buildOnyxDataForMultipleQuickbooksConfigurations<TConfigUpdate extends
     };
 }
 
+function updateQuickbooksOnlineAutoSync(policyID: string, settingValue: boolean) {
+    const optimisticData: OnyxUpdate[] = [
```

Comment:
NAB: What is the reason behind not using `buildOnyxDataForQuickbooksConfiguration` here?